simplivity/resources/policies.py

```diff
@@ -89,6 +92,32 @@ def create(self, name, flags=None, timeout=-1):
         affected_object = self._client.do_post(URL, data, timeout, flags=flags)[0]
         return self.get_by_id(affected_object["object_id"])
 
+    def resume(self, target=None, timeout=-1):
```

Comment:
Writing something like what I have pseudocode below will be much cleaner, and sustainable for the future.

Naturally, target.DATA_FIELD currently doesn't exist for the 3 supported object types, but it would be good to stick it into the class so that we know the target object type of the objects in the future.

```        method_url = "{}/resume".format(URL)
        data = {}

        if type(target) is in [Host, OmnistackCluster, ClusterGroup]:
            data["target_object_type" = target.DATA_FIELD
            data["target_object_id"] = target.data["id"]
        else:
            data["target_object_type"] = 'federation'

        self._client.do_post(method_url, data, timeout)
```

Reply:
Updated as per the suggestion